### DISCUSSION TITLE
Print Note doesn't work

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1582] [Q&A]
I was trying to print a PDF and when I choose Print Note from the note actions, nothing happens. Is that normal?

![image](https://user-images.githubusercontent.com/23059969/106043276-ffeac600-6092-11eb-832d-dfd525991805.png)

**Comment by zadam** [comment]
Hi, what platform do you use?

**Reply by hokutonanto** [reply]
Sorry, forgot to mention I'm on Windows 10 Pro

**Reply by zadam** [reply]
Is there any error in the developer console after you click on "print note"? (you can open it with `CTRL-SHIFT-I`)

**Reply by hokutonanto** [reply]
Yes, there are a few errors. These, in particular, show up right after I invoke the print command.
(I'm omitting the others that are present when I launch Trillium)

```
jquery.min.js:2 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.

DevTools failed to load SourceMap: Could not load content for http://127.0.0.1:37840/libraries/bootstrap/css/bootstrap.min.css.map: Unknown error
```

**Reply by zadam** [reply]
Hmm, those errors don't seem to be relevant (or important). 

Print note works by invoking an OS (Windows in this case) print dialog. Not sure where is the issue.
